### PR TITLE
added load button

### PR DESCRIPTION
**Pull request by EVPJ**

### CODE REVIEW

File: packages/web-app/src/components/StupidForm.tsx [hunk 1]
```diff
@@ -22,6 +23,7 @@ const CREATE_USER = gql`
 const useCreatePlayer = () =>
   useMutation<CreatePlayerMutation, MutationCreatePlayerArgs>(CREATE_USER);
 
+
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidForm.tsx:26]: delete whitespace
```suggestion
```

File: packages/web-app/src/components/StupidForm.tsx [hunk 2]
```diff
@@ -41,6 +43,25 @@ export const StupidForm = () => {
...
+    //         body: JSON.stringify( { id: id, name: name, lastName: lastName})
+
+    //     }).then((res) => console.log(res)) 
+    console.log("test")
+    //StupidPlayers()
+    window.location.reload();
+
+       
+  }, [createPlayer, username, password, balance]);
+
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidForm.tsx:64]: This code isn't being used, delete
```suggestion
```

File: packages/web-app/src/components/StupidForm.tsx [hunk 3]
```diff
@@ -68,6 +89,7 @@ export const StupidForm = () => {
         onChange={(e) => setBalance(parseInt(e.target.value))}
       ></input>
       <button onClick={onFormSubmit}>Submit</button>
+      {/* <button onClick={reloadNow}>load database</button> */}
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidForm.tsx:92]: unused component, delete
```suggestion
```

File: packages/web-app/src/components/StupidPlayers.tsx [hunk 4]
```diff
@@ -14,8 +14,12 @@ const USERS = gql`
 `;
 const usePlayers = () => useQuery<PlayersQuery>(USERS);
 
+//const useLazyPlayers = () => useLazyQuery<PlayersQuery>(USERS);
+
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidPlayers.tsx:18]: delete
```suggestion
```

File: packages/web-app/src/components/StupidPlayers.tsx [hunk 5]
```diff
@@ -14,8 +14,12 @@ const USERS = gql`
...
 export const StupidPlayers = () => {
-  const { data } = usePlayers();
+  const { data, refetch } = usePlayers();
+  //const [getPlayers, {data}] = useLazyPlayers();
+
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidPlayers.tsx:22]: delete 
```suggestion
```

File: packages/web-app/src/components/StupidPlayers.tsx [hunk 6]
```diff
@@ -24,8 +28,13 @@ export const StupidPlayers = () => {
           <span>Username: {username}</span>
           <span>Balance: {balance}</span>
           <span>Is Admin: {isAdmin}</span>
+
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidPlayers.tsx:31]: delete
```suggestion
```

File: packages/web-app/src/components/StupidPlayers.tsx [hunk 7]
```diff
@@ -24,8 +28,13 @@ export const StupidPlayers = () => {
           <span>Username: {username}</span>
           <span>Balance: {balance}</span>
           <span>Is Admin: {isAdmin}</span>
+          
         </div>
+
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidPlayers.tsx:33]: delete 
```suggestion
```

File: packages/web-app/src/components/StupidPlayers.tsx [hunk 8]
```diff
@@ -24,8 +28,13 @@ export const StupidPlayers = () => {
           <span>Username: {username}</span>
           <span>Balance: {balance}</span>
           <span>Is Admin: {isAdmin}</span>
+          
         </div>
+        
       ))}
+      <button onClick={
+        () => refetch()
+        }>refetch</button>
```
Comment (calebgmahala) [inline packages/web-app/src/components/StupidPlayers.tsx:37]: Linter should have fixed this, I will look into why it didn't and push change to fix this in the future. Will also add linting script to base of repo

**Review by calebgmahala — CHANGES_REQUESTED** [review]
delete unused stuff

**Comment by EVPJ** [comment]
re review

**Review by calebgmahala — APPROVED** [review]

**Pull request merged**